packages/hash/api/src/model/knowledge/page.model.ts

```diff
@@ -259,7 +259,7 @@ export default class extends EntityModel {
       return false;
     }
 
-    if (parentPage.entityId === page.entityId) {
+    if (parentPage.entityUuid === page.entityUuid) {
```

Comment:
Should we compare `baseId` here?

Reply:
Addressed as part of https://github.com/hashintel/hash/pull/1435/commits/3cb6ed29868d750d40108e418521ade0b2bf056f